components/nimbus/src/enrollment.rs

```diff
@@ -1445,6 +1445,88 @@ mod tests {
         .unwrap()
     }
 
+    fn get_conflicting_multi_feature_experiments() -> Vec<Experiment> {
+        serde_json::from_value(json!(
+            [
+            // This Experiment has three distinct features that a user can
+            // be involved in `non-conflicting-feature`, `conflicting-feature` and
+            // `some-non-conflicting-treatment`, any other experiment involving one of those
+            // would conflict with this experiment.
+            {
+            "schemaVersion": "1.0.0",
+            "slug": "another-monkey",
+            "endDate": null,
+            "branches":[
+                {"slug": "control", "ratio": 1, "features": [
+                    { "featureId": "non-conflicting-feature", "enabled": true },
+                    { "featureId": "conflicting-feature", "enabled": false}
+                ]},
+                {"slug": "treatment","ratio": 1, "features": [
+                    { "featureId": "some-non-conflicting-treatment", "enabled": true },
+                    { "featureId": "some-non-conflicting-treatment", "enabled": false },
```

Comment:
nit: indentation, and trailing comma after last item in list

Reply:
Calling this `some-non-conflicting-treatment` confused me:

1. `treatment` normally refers to a branch, not a feature.
2. once we get to about 3 of test/example anything, a simple naming scheme really helps understanding: if you don't like `feature-A`, `feature-B`, `feature-C` , there's always `alice-feature`, `bob-feature` and `charlie-feature` or `antelope`, `baboon`, `chimpanzee`.